tests/fakeapi/aiohttp_handlers.py

```diff
@@ -45,6 +45,21 @@ def aiohttp_access_request_context(request_ctx):
     return ConnexionResponse(status_code=204)
 
 
+@asyncio.coroutine
+def aiohttp_query_parsing_str(query):
+    return ConnexionResponse(body={'query': query})
```

Comment:
FYI you could just `return query`, but this is totally fine too.

Reply:
Yeah, I know I could've but just wanted to be consistent with the rest of the handlers here. I am pretty picky about this point thanks to my colleagues from FE team :D